src/library/Uno.Material/Styles/Controls/TextBox.xaml

```diff
@@ -5,6 +5,7 @@
 					xmlns:not_macos="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
 					xmlns:macos="http://uno.ui/macos"
 					xmlns:um="using:Uno.Material"
+					xmlns:controls="using:Uno.Material.Controls"
```

Comment:
`Ripple` is defined under `Uno.Material` without the `.Controls` part.
Also, `xmlns:controls` is already used for `using:Microsoft.UI.Xaml.Controls`. Leaving it here would cause XamlMerge to complaint. So you might want to remove it.